packages/lib-content/src/screens/Home/DefaultHome/components/Researchers.js

```diff
@@ -0,0 +1,126 @@
+import { Anchor, Box, Paragraph } from 'grommet'
+import { useTranslation } from '../../../../translations/i18n.js'
+import styled, { css } from 'styled-components'
+import { SpacedHeading } from '@zooniverse/react-components'
+
+import SubHeading from '../../../../components/HeadingForAboutNav/SubHeading.js'
+import MaxWidthContent from '../../../../components/MaxWidthContent/MaxWidthContent.js'
+
+const StyledLink = styled(Anchor)`
+  width: 300px;
+  border-radius: 5px;
+  font-size: 0.8rem;
+  padding: 8px 5px;
+  text-align: center;
+  color: black;
+  background-color: ${props => props.theme.global.colors['neutral-2']};
+  font-weight: normal;
+  box-shadow: 1px 1px 8px rgba(0, 0, 0, 0.25);
+
+  &:hover {
+    text-decoration: none;
+  }
+`
+
+const Relative = styled(Box)`
+  position: relative;
+`
+
+const OverlayFade = styled(Box)`
+  position: absolute;
+  right: -1px;
+  top: 0;
+  width: 30px;
+  height: 100%;
+  ${props =>
+    props.theme.dark
+      ? css`
+          background: linear-gradient(
+            90deg,
+            rgba(255, 255, 255, 0) 0%,
+            ${props.theme.global.colors['dark-3']} 100%
+          );
+        `
+      : css`
+          background: linear-gradient(
+            90deg,
+            rgba(255, 255, 255, 0) 0%,
+            #fff 100%
+          );
+        `}
+`
+
+const VideoWrapper = styled(Box)`
+  min-width: 478px;
+  border-radius: 8px;
+  aspect-ratio: 16 / 9;
+  overflow: hidden;
+
+  @media (width <= 769px) {
+    min-width: 370px;
+  }
+`
+
+const YOUTUBE_SOURCES = [
+  'https://www.youtube-nocookie.com/embed/CaTNIoJy4Dg?si=-rG8luvWD_FtFBc5', // PH TESS
```

Comment:
[rambling] This is my first time seeing `www.youtube-nocookie.com`. I tried searching to find more info on it, and found the official documentation _buried very low_ on the Google search results under YouTube Help -> Embed videos and playlists -> Turn on privacy-enhanced mode. _Fascinating._

<details>
<summary>For posterity, here's what's written on that YouTube "Embed videos and playlists" help page's "Turn on privacy-enhanced mode" section, as of 6 Jun 2024</summary>

The privacy-enhanced mode of the YouTube embedded player prevents the use of views of embedded YouTube content from influencing the viewer's browsing experience on YouTube. This means that the view of a video shown in the privacy-enhanced mode of the embedded player will not be used to personalise the YouTube browsing experience, either within your privacy-enhanced mode embedded player or in the viewer's subsequent YouTube viewing experience.

If ads are served on a video shown in the privacy-enhanced mode of the embedded player, those ads will likewise be non-personalised. In addition, the view of a video shown in the privacy-enhanced mode of the embedded player will not be used to personalise advertising shown to the viewer outside of your site or app.

As a reminder, the [YouTube API Terms of Service](https://developers.google.com/youtube/terms/api-services-terms-of-service) and [developer policies](https://developers.google.com/youtube/terms/developer-policies) apply to the access and use of the YouTube embedded player.

To use privacy-enhanced mode:

Change the domain for the embed URL in your HTML from https://www.youtube.com to https://www.youtube-nocookie.com.
For network administrators: You'll need to add youtube-nocookie.com to the firewall allowlist.
For use in applications, use a WebView instance of the embedded player. Privacy-enhanced mode is currently available only for embedded players on websites.
Important: If your website or app is child-directed, you must self-designate your site or app using [these tools](https://support.google.com/policies/answer/9664901), as required by the [YouTube API Terms of Service](https://developers.google.com/youtube/terms/api-services-terms-of-service) and [developer policies](https://developers.google.com/youtube/terms/developer-policies#j.-child-directed-api-clients), even if you embed YouTube videos with the privacy-enhanced mode player.
Example:

Before

<iframe width="1440" height="762"

src="https://www.youtube.com/embed/7cjVj1ZyzyE"

frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>

After

<iframe width="1440" height="762" src="https://www.youtube-nocookie.com/embed/7cjVj1ZyzyE"

frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
</details>

Reply:
This is good to know! I think when I grab video URLs to embed on the new homepage, I'm always using Brave so it's likely that browser auto-adds `no-cookie` 🤷‍♀️